src/docfx/build/xref/InternalXrefMapBuilder.cs

```diff
@@ -147,7 +147,7 @@ private InternalXrefSpec[] AggregateXrefSpecs(string uid, InternalXrefSpec[] spe
             // uid conflicts with overlapping monikers
```

Comment:
Check why line 137 has a `Where(item.Monikers.Count == 0)` filter and fix that instead of adding a new check

Reply:
Because we need to handle moniker related duplicates differently.

Reply:
log duplicate-uid warning when 2 or more spec have completely the same MonikerList